src/main/resources/FXML/Menu.fxml

```diff
@@ -77,12 +77,11 @@
                     <items>
                       <AnchorPane maxWidth="150.0" minHeight="-Infinity" minWidth="-Infinity" prefWidth="150.0">
                            <children>
-                              <TextField fx:id="nodeTextField" disable="true" alignment="CENTER" layoutX="27.0" layoutY="30.0" prefHeight="29.0" prefWidth="97.0" promptText="Centre node" />
-                              <Button fx:id="goToNodeBut" layoutX="26.0" layoutY="72.0" disable="true" mnemonicParsing="false" onAction="#traverseGraphClicked" text="Go to this node" />
-                              <TextField fx:id="radiusTextField" disable="true" alignment="CENTER" layoutX="27.0" layoutY="111.0" prefHeight="29.0" prefWidth="97.0" promptText="Radius" />
-                              <Button fx:id="chooseGenome" disable="true" layoutX="14.0" layoutY="171.0" mnemonicParsing="false" onAction="#chooseGenomePress" prefHeight="25.0" prefWidth="120.0" text="Specific Genome" />
-                              <Button fx:id="annoBut" disable="true" layoutX="12.0" layoutY="208.0" mnemonicParsing="false" onAction="#chooseAnnoClicked" prefHeight="25.0" prefWidth="120.0" text="Choose Annotation" />
-                              <CheckBox fx:id="rainbowBut" disable="true" layoutX="36.0" layoutY="255.0" mnemonicParsing="false" onAction="#rainbowButtonClicked" selected="true" text="Rainbow view" />
+                              <TextField fx:id="nodeTextField" alignment="CENTER" layoutX="27.0" layoutY="30.0" prefHeight="29.0" prefWidth="97.0" promptText="Centre node" />
```

Comment:
Because they are still enabled after a gfa file is loaded in the update method in MenuController right???